testfiles/TF06/06.2-all-fail-1.html

```diff
@@ -2,12 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Decorative Image - non-descriptive text in accessible name and description (TC06.2-all-fail-1)</title>
+    <title>Decorative Image - with no attribute to define accessible name (TC06.2-all-fail-1)</title>
```

Comment:
include the full description from the spreadsheet - "decorative image with no accessible name or description defined, not in background"